src/revert.ts

```diff
@@ -0,0 +1,89 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
I'm not clear on copyrights, but if this is an original file, you shouldn't be copyrighting bcrowl.

Reply:
okay, removed the copyright